### PR TITLE
PLANET-7217 Fix carousel header flicker issue

### PR DESCRIPTION
**Pull request by sagarsdeshmukh**
Revert some change from [PLANET-7189](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/commit/7e6724d44c39abc4b734e879e6dd6f4a75de54e0#) PR

**Testing:**
- You can test fix at - https://www-dev.greenpeace.org/test-telesto/test-carousel-header-block-fliker-image-issue/

[Demo page](https://www-dev.greenpeace.org/test-telesto/carousel-header-demo-page/) with the fix (editor and frontend), also tested on the [international dev site](https://github.com/greenpeace/planet4-international/blob/main/composer-local.json#L6).

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [telesto](https://www-dev.greenpeace.org/test-telesto) | [admin](https://www-dev.greenpeace.org/test-telesto/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-telesto/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-telesto) | [composer-local.json](https://github.com/greenpeace/planet4-test-telesto/blob/main/composer-local.json)

:watch: 2023.08.17 10:07:39

File: assets/src/blocks/CarouselHeader/CarouselControls.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 const {__} = wp.i18n;
 
 export const CarouselControls = ({
-  goToPrevSlide = null,
-  goToNextSlide = null,
-  goToSlide = null,
+  goToPrevSlide = () => {},
```
Comment (GP-Dan-Tovbein) [inline assets/src/blocks/CarouselHeader/CarouselControls.js:4]: Prefer to set as a `void` function

File: assets/src/blocks/CarouselHeader/CarouselHeaderEditor.js [hunk 2]
```diff
@@ -124,8 +124,8 @@ export const CarouselHeaderEditor = ({setAttributes, attributes}) => {
             </Slide>
           ))}
           <CarouselControls
-            goToPrevSlide={goToPrevSlide}
-            goToNextSlide={goToNextSlide}
+            goToPrevSlide={() => goToPrevSlide(carousel_autoplay)}
```
Comment (GP-Dan-Tovbein) [inline assets/src/blocks/CarouselHeader/CarouselHeaderEditor.js:127]: Previously it never received the `autoplay` value. It had been received the `event` as a parameter

**Review by comzeradd — APPROVED** [review]
💯

**Pull request merged**